pyproject.toml

```diff
@@ -54,15 +52,16 @@ dependencies = [
     "python-dateutil==2.8.2",
     "python-rapidjson>=1.9,<1.17",
     "pytz==2024.1",
-    "redis>=4.6, <4.7.0",
+    "redis==5.0.3",
     "schedule==1.1.0",
     "sentry-sdk[fastapi]==1.43.0",
     "SQLAlchemy==2.0.28",
     "SQLAlchemy-Utils==0.41.2",
     "structlog",
     "typer==0.9.0",
     "uvicorn[standard]~=0.28.0",
-    "nwa-stdlib~=1.6.0",
+#    "nwa-stdlib~=1.6.0",
+    "nwa-stdlib @ git+https://github.com/workfloworchestrator/nwa-stdlib@redis-py-v5",
```

Comment:
Update pin to 1.7.0 after release of https://github.com/workfloworchestrator/nwa-stdlib/pull/31